### DISCUSSION TITLE
Websites Coding

### DISCUSSION THREAD

**Discussion by nomanamin12** [community/community#71547] [Programming Help]
### Body

I am a programmer who wants to create a website, but I need some assistance.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by OSEight** [comment]
I can help!

**Reply by OSEight** [reply]
<!-- This is an invisible message! --!>

**Comment by deep100x** [comment]
i am ready to help anytime !! just contact me.

**Reply by ghost** [reply]
Hello Deep, could you help me with one of my [projects](https://github.com/lisztomania23/landing-page-for-space-agency-cosmezis/tree/main). 👈 [See preview of the page](https://lisztomania23.github.io/landing-page-for-space-agency-cosmezis/). 👈 In the ***[space odyssey section](https://github.com/lisztomania23/landing-page-for-space-agency-cosmezis/blob/main/style.css)***, I am unable to place the sphere image in the correct position ***for small screens***. I've tried multiple media queries, but it seems there might be an issue with the property and values in my CSS. Perhaps my CSS is not written in the best way. I would like to know how I can improve my CSS.

**Reply by deep100x** [reply]
hey bro really sorry for late can we discuss what you want to do !

**Reply by ghost** [reply]
@deepk2891 preview the page, decrease the screen width in dev tool you'll find the bug i'm talking about

**Reply by deep100x** [reply]
email me your insta or whatsapp we will definetely figure it out as soon as possible

**Comment by kur0neko** [comment]
I think the best way is you should also try to learn and understand by doing it.  I personally learn so much after doing some free lessons on https://www.freecodecamp.org/
